src/Command/Massban.js

```diff
@@ -0,0 +1,111 @@
+const Command = require("../Command");
+const RoleDeterminer = require("../Helper/RoleDeterminer");
+const writeToLog = require("../Helper/GenericLogger");
```

Comment:
This should be part of the dependency graph, not an in-file requirement.

Reason being: to make it mockable for unit testing

Reply:
None of the helpers in any of the commands are part of the dependency graph, but I guess I can add these ones